src/ffi/verona-ffi.cc

```diff
@@ -0,0 +1,257 @@
+// Copyright Microsoft and Project Verona Contributors.
+// SPDX-License-Identifier: MIT
+
+#include "CXXInterface.h"
+
+#include <fstream>
+#include <iomanip>
+
+using namespace std;
+using namespace verona::ffi;
+using namespace clang;
+namespace cl = llvm::cl;
+
+namespace
+{
+  // Test function (TODO: make this more generic)
```

Comment:
I'd suggest the opposite.  The amount of logic required to pull out a handful of configuration things for testing here is significant, why not just make these individual unit test binaries?  We'll want some (many!) integration tests later that generate LLVM IR from Verona programs that use C++ types, I'm not convinced that it's worth building a generic test program here.

Reply:
Right now, this is a hack in either direction and is just here as a "user"/"driver" of the internal API. This is neither a generic program nor a test file. Given this won't be used externally, there's little gain from having a truly generic "program", but not a more generic API.

I want to create proper unit-tests for these, but tests that express how we'll use the API from the sandbox side, which will need a more polished interface than we currently have. For example, I want to stop exposing the internal fields of `CXXInterface` and `Compiler` to the user and provide builders and helpers.

This will not only help writing tests that clearly mark the semantics and usability we need, but also simplify the sandbox code when creating the functions and querying the types it needs.

Right now, though, this is just a hack to show the API works in some way.